scio-examples/src/it/scala/com/spotify/scio/examples/RunPreReleaseIT.scala

```diff
@@ -0,0 +1,96 @@
+package com.spotify.scio.examples
+
+import com.spotify.scio.ContextAndArgs
+import com.spotify.scio.examples.extra.{AvroExample, ParquetExample}
+import com.spotify.scio.util.ScioUtil
+import org.slf4j.LoggerFactory
+
+import java.util.concurrent.TimeUnit
+import scala.concurrent.duration.Duration
+import scala.concurrent.{Await, ExecutionContext, Future}
+import scala.reflect.ClassTag
+import scala.util.{Failure, Success, Try}
+
+object RunPreReleaseIT {
+
+  implicit private val ec: ExecutionContext = ExecutionContext.global
+
+  private val defaultArgs = Array(
+    "--runner=DataflowRunner",
+    "--project=data-integration-test",
+    "--region=europe-west1",
+    "--tempLocation=gs://dataflow-tmp-europe-west1/gha"
+  )
+
+  private val log = LoggerFactory.getLogger(getClass)
+
+  def main(cmdLineArgs: Array[String]): Unit = {
+    val (_, args) = ContextAndArgs(cmdLineArgs)
+    val runId = args("runId")
+
+    Await.result(
+      Future
+        .sequence(
+          parquet(runId) :+ avro(runId)
+        )
+        .map(_ => log.info("All Dataflow jobs ran successfully."))
+        .recover(e => throw new RuntimeException("At least one Dataflow job failed", e)),
+      Duration(1, TimeUnit.HOURS)
+    )
+  }
+
+  private def avro(runId: String): Future[Unit] = {
```

Comment:
Minor thing, more like a nitpick. Maybe it's better to return `List` even now when `avro` has only one job? It will create an implicit job execution interface `String => List[Future[Unit]]` so all primitive job executions could be uniformly composed into a single list of futures?

Reply:
good idea! I'll update that.